doc/api.md

```diff
@@ -2,31 +2,89 @@
 
 ***If you are running the server and want to enable the API, set the environment variable `API_ENABLED=true`.***
 
-This API is currently in a very limited alpha, and is not enabled in any of our deployments. Note that this API may change at any time, without warning.
+The API design is evolving. Backwards-incompatible changes will result in a version change.
```

Comment:
since we essentially deleted v1, should we mention that here and and/or confirm that we will not be removing future versions similarly (don't think it's bad that we removed v1, but in the future I imagine we will want to continue to support deployed older API versions)